faunadb-scala/src/main/scala/faunadb/query/package.scala

```diff
@@ -696,6 +696,14 @@ package object query {
   def HasIdentity(): Expr =
     Expr(ObjectV("has_identity" -> NullV))
 
+  /**
+    * An HasCurrentToken expression.
```

Comment:
I _think_ it should be "A HasCurrentToken", but an English native speaker could confirm better than me! :)

Reply:
This is correct